requirements/base.txt # includes all dependencies in `base.txt

```diff
@@ -0,0 +1,14 @@
+# common dependencies used for all environments
```

Comment:
This file should not be added.

Reply:
@brylie, could you clarify that? 

Since we have Django 3.2.8, it is written that `pyup.io/safety-ci Django 3.2.8 has known security vulnerabilities.`, and we get` Some checks were not successful`. So we should update Django to the newest versions. Actually, I've not yet checked` Django 4.0` requires too many changes. If not, we can update Django. Or do you have any chance to look at the newer version?

Reply:
@brylie, could you check my previous message? 😊  I talked about Django 4.0. Instead, we can update to Django 3.2.10 first.

Reply:
The file name "requirements/base.txt # includes all dependencies in \`base.txt" seems like an error. Rather, the file name should be "requirements/base.txt" which already exists. This "requirements/base.txt # includes all dependencies in \`base.txt" file should not be included in the project.

Reply:
You're careful. :) I missed that one. 😅 So we can update Django to 3.2.10 after fixing this issue. Right?

Reply:
Yes, or proceed with Django 4 upgrade if it isn't too difficult, as you mention above.

Reply:
Ok, let's proceed with upgrades step by step. In the end, we improve to Django 4.

Reply:
Looks like Django 3.2.10 has also security vulnerabilities.